DfE.FindInformationAcademiesTrusts/Pages/Trusts/TrustsAreaModel.cs

```diff
@@ -41,21 +40,22 @@ public string MapDataSourceToName(DataSource source)
             case Source.ExploreEducationStatistics:
                 return "Explore education statistics";
             default:
-                _logger.LogError("Data source {source} does not map to known type", source);
+                logger.LogError("Data source {source} does not map to known type", source);
                 return "Unknown";
         }
     }
 
     public virtual async Task<IActionResult> OnGetAsync()
     {
-        var trust = await _trustProvider.GetTrustByUidAsync(Uid);
+        var trustSummaryDto = await TrustService.GetTrustSummaryAsync(Uid);
```

Comment:
Could we update the naming for consistency

Reply:
Done